Makefile

```
@@ -0,0 +1,35 @@
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+BINARY_NAME=main
```

```suggestion
BINARY_NAME=auth-gateway
```

And add a .gitgnore that excludes that file